DistanceMatrixApiClient.java

```diff
@@ -0,0 +1,49 @@
+package org.example;
+
+import org.json.JSONObject;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Scanner;
+
+public class DistanceMatrixApiClient {
+    private static final String API_KEY = "AIzaSyB2Gy1rH3_qM4nbYWwNoCB1AIcS4yoLjoI";

```

Comment:
try not to Include the real Api key when pushing the code online 
Its supposed to be a secret code